detection-rules/attachment_pdf_invoice.yml

```
@@ -0,0 +1,31 @@
+name: "Attachment: PDF Invoice from first-time sender"
+description: |
+  Detect potential fraudulent invoicing threats in pdf files by analyzing OCR'd text.
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+
+  and (
+          (
+             sender.email.domain.root_domain in $free_email_providers
+              and sender.email.email not in $sender_emails
+          )
+          or (
+              sender.email.domain.root_domain not in $free_email_providers
+              and sender.email.domain.domain not in $sender_domains
+          )
+      )
+  and (
+      any(attachments, .file_extension in~ ("pdf") and
+          any(beta.binexplode(.),
+          any(beta.ml_nlu_classifier(.scan.ocr.raw).tags,
+              .name in ("invoice", "payment")) and
+              any(beta.ml_nlu_classifier(.scan.ocr.raw).entities, .name == "org"))
+      )
+  )
+tags:
+  - "Machine Learning"
+  - "Natural Language Understanding"
+  - "Suspicious PDF"
+  - "Fraudulent Invoicing"
```

```suggestion
  - "Invoice fraud"
```
we use this tag in other rules currently